FSI/Stress.H

```diff
@@ -0,0 +1,67 @@
+#ifndef FSI_STRESS_H
+#define FSI_STRESS_H
+
+#include "CouplingDataUser.H"
+#include "Force.H"
+
+#include "fvCFD.H"
+
+// TEMPORARY
+#include "pointFields.H"
+#include "vectorField.H"
+#include "immiscibleIncompressibleTwoPhaseMixture.H"
```

Comment:
Why is this needed? Looks too specific.

Reply:
Copied from the forces module.

Reply:
Still, let's check if it is actually needed and remove it if not. Or at least remove the `// TEMPORARY`.

Reply:
Ok this is needed in the calculation of `mu` in case of mixture,  I will remove the temporary